.env.example

```diff
@@ -257,3 +257,5 @@ E2E_TEST_OIDC_USER_EMAIL=
 E2E_TEST_OIDC_USER_PASSWORD=
 
 # ***********************************************************************************************************
+
+ICAL_UID_DOMAIN="cal.com"
```

Comment:
We can probably reuse one of these https://github.com/calcom/cal.com/blob/0091ecf572c063e899ec664e2e3f6aa6e1c6499c/.env.example#L192 instead of adding a new env variable.